contents/node/nodejs.md

```diff
@@ -198,17 +218,15 @@ const newParser = new Parser();
 newParser.parse(content);
 ```
 
-To better understand how to use `module.exports`, checkout [Tendai Mutunhire's article](http://stackabuse.com/how-to-use-module-exports-in-node-js/).
-
-## Use cases
+## Use Cases
 
 Node is **good** for
-* **Processing high volumes of IO-bound requests**. A single instance of a Node server will be more efficient and can serve more requests with the same hardware than other usual servers (such as those written in Ruby on Rails). This makes the Node server faster and more scalable.
-* **Real time apps** where you have to process a large volume of requests with little delay. This includes instant messaging apps and collaborative editing apps where you can watch the document being modified live such as Trello and Google Docs. Node is a good choice as it can handle multiple client requests (even while waiting for responses).
-* **Single-page apps** where a lot of processing and rendering is done on the client's side and the backend server only needs to provide a simple API. Node can process many requests with low response times. In addition, you can reuse and share code, such as those for validation, between the client and server.
+* **Processing high volumes of I/O-bound requests**. A single instance of a Node server will be more efficient and can serve more requests with the same hardware than most other servers (such as those written in Ruby on Rails). This makes the Node server faster and more scalable.
+* **Real time applications** where you have to process a large volume of requests with little delay. This includes instant messaging apps and collaborative editing apps where you can watch the document being modified live such as Trello and Google Docs. Node is a good choice as it can handle multiple client requests even while waiting for responses.
+* **Single-page applications** where a lot of processing and rendering is done on the client's side and the backend server only needs to provide a simple API. Node can process many requests with low response times. In addition, you can reuse and share code, such as those for validation, between the client and server.
 
 However, Node is **not suitable** for
-* **CPU-intensive jobs**. Remember that Node is single-threaded. If the thread is busy doing CPU-heavy operations, it will not be able to process incoming requests timely. This removes the biggest advantage of Node. Perhaps, you should consider an alternative technology such as Go which supports concurrency.
+* **CPU-intensive jobs**. Recall that Node is single-threaded. If the thread is busy doing CPU-heavy operations, it will not be able to process incoming requests timely. This removes the biggest advantage of Node. In such cases, you could consider alternatives such as Go which supports concurrency.
```

Comment:
"concurrency" technically means executing processes in a (pseudo-)parallel manner and Node supports concurrency by nature. I guess the author meant to say that Node is after all a JavaScript runtime that's not as performant as a native compiled language like Go, and CPU-intensive tasks won't execute as efficiently in Node as Go.